### PR TITLE
login and scope pages

### PR DESCRIPTION
**Pull request by sash-257defi**

### CODE REVIEW

**Review by ashkanx — CHANGES_REQUESTED** [review]
This should be in SASS format.

**Comment by sash-257defi** [comment]
Fixed

File: _sass/common.scss [hunk 1]
```diff
@@ -141,3 +141,23 @@ header,
 .invisible {
     display: none !important;
 }
+//login scope
+.fx-acc {
```
Comment (negarn) [inline _sass/common.scss:145]: @sashcy seems this element doesn't exist in oauth page, are you planning to create a separate PR for that to add this element? or is it better to use the existing elements?

Comment (sash-257defi) [reply _sass/common.scss:145]: No, I sent new pages for oath (login and scope) where fx-login is renamed to fx-acc so it can be reused.

Comment (ashkanx) [reply _sass/common.scss:145]: Sash, can you send the HTML of both pages to me? I can't find them in my email.

**Review by ashkanx — APPROVED** [review]

**Pull request merged**